### PR TITLE
fix condition on default outcome

### LINKED ISSUE #2985: Error when user directly to market trading page
image

### PR DESCRIPTION
**Pull request by bthaile**
fixes https://github.com/AugurProject/augur/issues/2985

fixed issue with conditional. need to test for null and undefined.

### CODE REVIEW

File: packages/augur-ui/src/modules/market/components/market-view/market-view.tsx [hunk 1]
```diff
@@ -265,7 +265,7 @@ export default class MarketView extends Component<MarketViewProps, MarketViewSta
       );
     }
 
-    let outcomeId = selectedOutcomeId !== null ? selectedOutcomeId : market.defaultSelectedOutcomeId;
+    let outcomeId = selectedOutcomeId ? selectedOutcomeId : market.defaultSelectedOutcomeId;
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/market/components/market-view/market-view.tsx]: I think we will need to test specifically test for NULL UNDEFINED here

**Review by andrewdamelio — APPROVED** [review]
